### PR TITLE
fix: Changed the UI and functionality of buttons in Lux Meter

### LINKED ISSUE #1284: Confusing buttons in Lux Meter instrument
**Actual Behaviour**

The buttons to record data and show data have confusing icons

**Expected Behaviour**

The icons should be informative

**Steps to reproduce it**

Observe Lux Meter instrument

**LogCat for the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by harsh-2711**
Fixes #1284 

**Changes**:

- Changed the label in the BottomNavigationMenu

- Changed the saved icon to record and pause icon

- Added delete and export data features

**Screenshot/s for the changes**:
[confusingBtns.zip](https://github.com/fossasia/pslab-android/files/2238372/confusingBtns.zip)

The updated icons screenshot: (@harsh-2711 please resize screenshots in PRs, I've done it here)

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2242186/app-debug.zip)

### CODE REVIEW

File: app/src/main/java/org/pslab/activity/LuxMeterActivity.java [hunk 1]
```diff
@@ -193,38 +199,68 @@ public boolean onTouchEvent(MotionEvent event) {
...
                         gpsLogger.startFetchingLocation();
                     } else {
-                        saveData = true;
+                        recordData = true;
                         CustomSnackBar.showSnackBar(coordinatorLayout, getString(R.string.data_recording_start) + "\n" + getString(R.string.location_disabled), null, null);
                     }
                 }
-                invalidateOptionsMenu();
+                break;
+            case R.id.save_csv_data:
```
Comment (CloudyPadmal) [inline app/src/main/java/org/pslab/activity/LuxMeterActivity.java]: Change this to record_csv_data as it is changed in all other places

**Comment by harsh-2711** [comment]
@CloudyPadmal All changes are done. Please review.

File: app/src/main/res/drawable/delete_icon.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0"
+        android:tint="?attr/colorControlNormal">
+    <path
+        android:fillColor="#FFF"
```
Comment (harsh-2711) [inline app/src/main/res/drawable/delete_icon.xml]: The color is hard coded because referring to resources file from this file isn't supported on API <21.

**Comment by Avjeet** [comment]
icon is shpwing properly 
image
<img src = "https://user-images.githubusercontent.com/20573611/43424020-a8780cf2-946b-11e8-8c2b-d74612cd402d.png" width = "200" height = "100"

**Comment by harsh-2711** [comment]
@CloudyPadmal I have used vector drawables only. You can check the extension of the file it's .xml. I have also updated the apk. Have a look.

File: app/src/main/res/drawable/record_icon.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0"
+        android:tint="?attr/colorControlNormal">
```
Comment (CloudyPadmal) [inline app/src/main/res/drawable/record_icon.xml]: Applying tint attributes shows a warning
> Resource references will not work correctly in images generated for this vector icon for API < 21

Current min SDK is 16. Please remove this tint attribute from icons in this PR.

**Comment by Avjeet** [comment]
@harsh-2711 In this PR, even If I don't click on **Export data** the CSV file is still getting created in my external storage, if that is the case then what is point of having export data button.

**Comment by Avjeet** [comment]
@harsh-2711 Can you make this PR only concerned with UI and remove the play/pause functionalities because I created them in my PR.

**Comment by Avjeet** [comment]
@harsh-2711 what is the update on this?

**Comment by harsh-2711** [comment]
@Avjeet Only changing the buttons makes no sense. I have completed working on the issue of the file being created. I will soon update the PR. Also, I will change the title of this PR.

**Comment by harsh-2711** [comment]
@Avjeet @CloudyPadmal I have made the suggested changes. Please review. I have also made this suggested changes in PR #1320. Good catch @Avjeet

File: app/src/main/res/drawable/record_icon.xml (hunk 3 above)
Comment (harsh-2711) [reply app/src/main/res/drawable/record_icon.xml]: Sure @CloudyPadmal

**Comment by harsh-2711** [comment]
@Avjeet A user opens an instrument only when he wants to use it. Therefore, it's not logical to freeze the instrument until he clicks the play button. In this PR, the user will be able to use the instrument as soon as he opens it and will able to record the data if he wants. Also, for the database issue, I can store the data first in a database and then in a file when a user clicks Export Data if you want (Current;). Else you can add this functionality in your PR #1323. And I don't think there's anything difficult for a user to understand how to use the buttons in the instrument as I have used the icons from Material Design library of Google.

**Comment by Avjeet** [comment]
@harsh-2711 In last meeting it has been discussed there is no need of delete and export button here in lux meter it will be shifted to logged data activity and also could you try to make a icon like this 
image

**Comment by Avjeet** [comment]
@CloudyPadmal @harsh-2711 @cweitat I think the changes from this PR also got merged into the code with the PR #1333

**Comment by harsh-2711** [comment]
@Avjeet  I did a rebase and reset so all the commits got squashed. But I did made a note about that in the PR body. I think @cweitat missed it.

**Comment by Avjeet** [comment]
@harsh-2711 Ok harsh I will make the new icon in my PR pls review it

**Pull request closed without merging**